src/networks.json

```diff
@@ -893,6 +893,21 @@
     "start": 859041,
     "logo": "ipfs://QmeGbNTU2Jqwg8qLTMGW8n8HSi2VdgCncAaeGzLx6gYnD7"
   },
+  "1319": {
+    "key": "1319",
+    "name": "AITDChain Mainnet",
+    "shortName": "AITD",
+    "chainId": 1319,
+    "network": "mainnet",
+    "testnet": false,
+    "multicall": "0x16e98974D30263252aa273dC8e3d1d48380845FB",
+    "rpc": [
+      "http://node.aitd.io"
```

Comment:
Hi @xuzuguang This RPC doesn't work, i get CORS issues,http://walletrpc.aitd.io but it is not a full archive node